tests/elf/Makefile

```diff
@@ -9,9 +9,9 @@ SOURCES = $(wildcard *.c)
 
 INCLUDES = -I$(SUBOBJDIR) -I$(INCDIR)
 
-CFLAGS = $(OEHOST_CFLAGS)
+CFLAGS = $(OEHOST_CFLAGS) $(GCOV_CFLAGS)
 
-LDFLAGS = $(OEHOST_LDFLAGS)
+LDFLAGS = $(OEHOST_LDFLAGS) $(GCOV_LDFLAGS)
```

Comment:
I don't think `GCOV_LDFLAGS` is ever defined.

Reply:
Removed it entirely, seems like it was used in the previous revision